### ISSUE TITLE
external app player, cause the streamplayer to stopworking and can`t resuming

### ISSUE THREAD

**Issue by syrakozz** [tumtumtum/StreamingKit#178]
external app player, cause the streamplayer to stopworking and can`t resuming

**Comment by francosolerio** [comment]
Your app should respond to AVAudioSessionInterruptionNotification and pause/resume the player accordingly.

**Comment by syrakozz** [comment]
you mean i should write this code in the appdelegate   ?
http://stackoverflow.com/a/20630994/987407

**Comment by francosolerio** [comment]
Yes, that looks good. You have to add code to pause and resume STKAudioPlayer in the notification handling methods. 
Stack Overflow is great for asking help and info about this issues. Better use  GitHub issues for bugs and pull requests.

**Comment by syrakozz** [comment]
great, thank you, 
i wish if @tumtumtum  can update the example project with the correct use for these general  issue

**Comment by tumtumtum** [comment]
STK is fairy low level and a easier API for common use cases is needed. I hear you guys. Been a bit busy this last month. I will get there.

**Comment by syrakozz** [comment]
great, thank you :)

**Comment by syrakozz** [comment]
i added, and i tested i pause the player on interruption , 
but the main problem is still exist, 
you where playing and some and then you call someone, and return to the player,
the player stop working, he only can fetch the duration of the song but he can`t play, i need to restart the app so he can play again , 
plz help

**Comment by francosolerio** [comment]
Can you put some example project for reproducing this bug?
